libs/perun/components/src/lib/edit-group-dialog/edit-group-dialog.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="{{theme}}">
+  <h1 mat-dialog-title>{{'DIALOGS.EDIT_GROUP.TITLE' | translate}}</h1>
+  <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+  <div *ngIf="!loading">
+    <div class="col pr-0" mat-dialog-content>
+      <mat-form-field class="w-100">
+        <label class="w-100">
+          <input [formControl]="nameControl" matInput placeholder="{{'DIALOGS.EDIT_GROUP.NAME' | translate}}">
+        </label>
+      </mat-form-field>
+      <mat-form-field class="w-100">
+        <label class="w-100">
+          <textarea [formControl]="descriptionControl" class="overflow-hidden" mat-autosize matInput placeholder="{{'DIALOGS.EDIT_GROUP.DESCRIPTION' | translate}}"></textarea>
+        </label>
+      </mat-form-field>
```

Comment:
There is no mat-error message when the name/description of the group is empty.
Also I think there would also be needed things from your other PR where there should be some message about group name regex.